src/DI/NavigationPanelExtension.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Tracy\DI;
+
+use Contributte\Tracy\NavigationPanel;
+use Nette\Application\IPresenterFactory;
+use Nette\Application\LinkGenerator;
+use Nette\Application\UI\Presenter;
+use Nette\DI\CompilerExtension;
+use Nette\DI\Statement;
+
+class NavigationPanelExtension extends CompilerExtension
+{
+
+	/** @var mixed[] */
+	private $defaults = [];
+
+	public function loadConfiguration(): void
+	{
+		$this->validateConfig($this->defaults);
+	}
+
+	public function beforeCompile(): void
+	{
+		$builder = $this->getContainerBuilder();
+
+		$linkGenerator = '@' . $builder->getByType(LinkGenerator::class);
```

Comment:
Did you try `builder->getDefinitionByType`? You can pass it directly into statement (line 37)